tests/Endatix.Api.Tests/Endpoints/Forms/UpdateTests.cs

```diff
@@ -0,0 +1,132 @@
+using FastEndpoints;
+using MediatR;
+using Microsoft.AspNetCore.Http.HttpResults;
+using Endatix.Core.Infrastructure.Result;
+using Endatix.Core.Entities;
+using Endatix.Api.Endpoints.Forms;
+using Endatix.Core.UseCases.Forms.Update;
+
+namespace Endatix.Api.Tests.Endpoints.Forms;
+
+public class UpdateTests
+{
+    private readonly IMediator _mediator;
+    private readonly Update _endpoint;
+
+    public UpdateTests()
+    {
+        _mediator = Substitute.For<IMediator>();
+        _endpoint = Factory.Create<Update>(_mediator);
+    }
+
+    [Fact]
+    public async Task ExecuteAsync_InvalidRequest_ReturnsBadRequest()
+    {
+        // Arrange
+        var formId = 1L;
+        var request = new UpdateFormRequest
+        {
+            FormId = formId,
+            Name = "Updated Form",
+            Description = "Updated Description",
+            IsEnabled = true
+        };
+        var result = Result.Invalid();
+        
+        _mediator.Send(Arg.Any<UpdateFormCommand>(), Arg.Any<CancellationToken>())
```

Comment:
No action required. Just a problem I didn't expect to happen.

Here you should be able to just pass the `request` instead of `Arg.Any<UpdateFormCommand>()` since the request is value object. 

However, even that we use a record type this doesn't work. That's weird and I will investigate at some point